Sources/HaCWebsiteLib/ViewModels/NotFound.swift

```diff
@@ -0,0 +1,27 @@
+import HaCTML
+
+struct NotFound {
+  
+  var node: Node {
+    return UI.Pages.base(
+      title: "Page Not Found",
+      content: Fragment(
+        El.Div[Attr.className => "NotFound_container"].containing(
```

Comment:
I am so glad to see you using HaCTML here very fluently. If you ran into anything confusing on the journey, could you let us know?